client/scss/application.scss

```diff
@@ -5,6 +5,7 @@
 @import 'utilities/variables';
 @import 'utilities/mixins';
 @import 'utilities/functions';
+@import 'utilities/classes';
```

Comment:
I think it's OK – the other three files in here don't contain classes (variables, mixins and functions).